### ISSUE TITLE
Help pages

### ISSUE THREAD

**Issue by AckslD** [glepnir/lspsaga.nvim#146]
**Is your feature request related to a problem? Please describe.**
Lookup documentation about lspsaga

**Describe the solution you'd like**
Write help pages so that one can do `:help lspsaga`

**Describe alternatives you've considered**
None

**Additional context**

**Comment by glepnir** [comment]
hmm will create it when i have time :)

**Comment by AckslD** [comment]
@glepnir Of course! Just thought it would be good to have an issue to track :) Thanks for the awesome plugin! :rocket:
